projects/systelab-preferences/tsconfig.lib.prod.json

```diff
@@ -1,6 +1,6 @@
 {
   "extends": "./tsconfig.lib.json",
   "angularCompilerOptions": {
-    "enableIvy": false
+    "compilationMode": "partial"
```

Comment:
ok. Oscar has just showed me that the recommendation is to use "partial Ivy" which goes with this configuration. Refer to https://angular.io/guide/creating-libraries#building-libraries-with-ivy